plugins/linear-genome-view/src/LinearGenomeView/model.ts

```diff
@@ -1516,6 +1516,31 @@ export function stateModelFactory(pluginManager: PluginManager) {
           : undefined
       },
     }))
+    .actions(self => ({
+      afterCreate() {
+        document.addEventListener('keydown', e => {
+          const session = getSession(self)
+          if (session.focusedView?.type === 'LinearGenomeView') {
```

Comment:
maybe the code can change to be like "if the session.focusedView === self, then do these operations on self" rather calling session.focusedView.##actionName"